### PR TITLE
Add "Poly Alias" - a new polyphonic example synth

### PR DESCRIPTION
**Pull request by 0xEAB**
I've created a new example synth :blush:
(well, in fact I've extracted the code from previous personal creations and refactored it).

It's name explains its functionality pretty well: it's polyphonic and its sound aliases like hot cakes. Furthermore, it features 3 raw digital waveforms (saw, square, sine) that can be changed via its only automation parameter. So it doesn't have a GUI.

If one doesn't demand overly high standards of it, it could be a solid base one can build a real synth plugin on. Basically it contains all the required "boilerplate" and "framework", but still misses the fun parts of synthesizer programming.

Please don't take the bundled [readme](https://github.com/0xEAB/dplug/blob/d03d6898adff38e94ab418f497b0973569f2d601/examples/poly-alias-synth/README.md) too serious (as written at the end of its text, it's a parody on grossly overstated product descriptions). If you think it oversteps the mark, I won't mind having it removed.

### CODE REVIEW

File: examples/poly-alias-synth/poly-alias-logo.svg [hunk 1]
```diff
@@ -1,10 +1,6 @@
...
 <svg width="700" height="70" xmlns="http://www.w3.org/2000/svg">
-    <g>
-        <title>Poly Alias - Open Source Example Synth</title>
-        <g display="none" overflow="visible" y="0" x="0" height="100%" width="100%" id="canvasGrid">
-            <rect fill="url(#gridpattern)" stroke-width="0" y="0" x="0" height="100%" width="100%"/>
-        </g>
-    </g>
     <g>
         <line x1="0" y1="0" x2="700" y2="0" stroke-width="4" stroke="#000000"/>
         <line x1="0" y1="70" x2="700" y2="70" stroke-width="4" stroke="#000000"/>
```
Comment (0xEAB) [inline examples/poly-alias-synth/poly-alias-logo.svg]: W3C validator says the SVG is fine, my browser renders it.
Not sure, what's GitHub's custom renderer's problem with it is.

Comment (0xEAB) [reply examples/poly-alias-synth/poly-alias-logo.svg]: imho the image is fine: [see there](https://github.com/0xEAB/dplug/blob/6c01baf366ce5baa16dff4a661559daeb184365c/examples/poly-alias-synth/README.md)

File: examples/poly-alias-synth/poly-alias-logo.svg [hunk 2]
```diff
@@ -1,10 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN" "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
 <svg width="700" height="70" xmlns="http://www.w3.org/2000/svg">
-    <g>
-        <title>Poly Alias - Open Source Example Synth</title>
-        <g display="none" overflow="visible" y="0" x="0" height="100%" width="100%" id="canvasGrid">
-            <rect fill="url(#gridpattern)" stroke-width="0" y="0" x="0" height="100%" width="100%"/>
-        </g>
-    </g>
```
Comment (0xEAB) [inline examples/poly-alias-synth/poly-alias-logo.svg]: this was just unnecessary bloat :see_no_evil:

File: examples/poly-alias-synth/poly-alias-logo.svg (hunk 1 above)
Comment (0xEAB) [reply examples/poly-alias-synth/poly-alias-logo.svg]: nautilus (file manager) doesn't neither complain. thumbnails works fine

**Comment by 0xEAB** [comment]
^ outdated notes aren't outdated

**Comment by p0nce** [comment]
LOL
The example are supposed to be ironic so it's fair game!

**Pull request merged**